crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -210,14 +210,16 @@ impl Plugin for VisibilityPlugin {
     fn build(&self, app: &mut bevy_app::App) {
         use VisibilitySystems::*;
 
-        app.add_system_to_stage(
-            CoreStage::PostUpdate,
-            calculate_bounds.label(CalculateBounds).before_commands(),
+        app.add_system(
+            calculate_bounds
+                .in_set(CalculateBounds)
+                // The commands from this system must be applied before we compute culling frustra
+                .in_set(CoreSet::Update),
```

Comment:
This should be reverted.

> calculate_bounds was in postupdate because people will add new assets / spawn entities using them in update, i imagine? so then adding an aabb if one does not exist was being done after that. what are the practical implications of moving it from CoreSet::PostUpdate to CoreSet::Update?

Reply:
Yeah this should definitely be reverted before merging. Discovered this yesterday while implementing base sets. Making it float around in Update will create weird off-by-one frame errors. PostUpdate is the right place for it. Pretty sure we just need to add an "apply buffers" between it and the systems that depend on its outputs (Ex: put an apply_buffers in CalculateBounds ordered after calculate_bounds, then find every internal system that depends on the Aabb component and add an ordering constraint).